rezervo/providers/sats/urls.py

```diff
@@ -0,0 +1,17 @@
+BASE_PATH = "https://www.sats.no"
+
+LOGIN_PATH = "/logg-inn"
+AUTH_PATH = "/api/log-in"
+BOOK_PATH = "/api/book"
+UNBOOK_PATH = "/api/unbook"
```

Comment:
Can we use `CANCEL_BOOKING_PATH` instead?

Reply:
Implemented in https://github.com/mathiazom/rezervo/pull/43/commits/4ab071ef60185945e9c1fe7cd396e4174c4f9232